### DISCUSSION TITLE
Migrating DB From Netbox Docker To Natively Hosted

### DISCUSSION THREAD

**Discussion by KSBolton** [netbox-community/netbox-docker#484] [Getting Help] [answered]
Hello all. I've been using the Docker image to basically test out Netbox and I'd like to move forward with it in a production environment. I was advised it would be best to run the application natively so I've installed it without Docker but now I'm wondering how best I could migrate the DB from Docker instance to native instance. 

I was thinking a simple export (via "Take a database backup" https://github.com/netbox-community/netbox-docker/wiki/Troubleshooting) and then import (via "Load an Exported Database" https://netbox.readthedocs.io/en/stable/administration/replicating-netbox/) would be enough. Would that work for what I'm trying to achieve?

**Comment by KSBolton** [comment, marked as answer]
Seems to work fine so far - had to change my DB username and password to match what was in the env variables for the Docker instance. That held me up for a while. I guess you can mark this as closed.

**Reply by cimnine** [reply]
If you have uploaded any media file, don't forget to migrate these files as well.

**Reply by KSBolton** [reply]
Ah, thank you! I missed those.
Just to be clear - is best practice for a production environment to run Netbox natively on a machine vs in a Docker container? I see you're a maintainer so I'm thinking you're the best person to ask, in case my team asks me why I moved the environment.
